wtforms_appengine/ndb.py

```diff
@@ -358,6 +360,9 @@ def convert_KeyProperty(self, model, prop, kwargs):
         else:
             return KeyPropertyField(**kwargs)
 
+    def convert__ClassKeyProperty(self, model, prop, kwargs):
```

Comment:
This bit is fine with me...but I'd prefer to only update the version number when we do an actual release to NPM.